modules/sony/display/bravia.rb

```diff
@@ -0,0 +1,234 @@
+# frozen_string_literal: true
+# encoding: ASCII-8BIT
+
+module Sony; end
+module Sony::Display; end
+
+# Documentation: https://aca.im/driver_docs/Sony/sony+bravia+simple+ip+control.pdf
+
+class Sony::Display::Bravia
+    include ::Orchestrator::Constants
+
+    # Discovery Information
+    tcp_port 20060
+    descriptive_name 'Sony Bravia LCD Display'
+    generic_name :Display
+
+    # Communication settings
+    # 24bytes with header however we'll ignore the footer
+    tokenize indicator: "\x2A\x53", msg_length: 21
+
+    def on_load # :nodoc:
```

Comment:
Docgen is currently handled by YARD so no `:nodoc:` directives :cry: 

For anything to be kept internal it should be possible to flag it with a `@!visibility private` as I've setup this to be left out of generated docs.

Reply:
I'm good with whatever you think - I never really got around to generating docs and yard was all the rage at some point